flink-python/pyflink/testing/test_case_utils.py

```diff
@@ -149,6 +149,30 @@ def collect(self, table):
         return string_result
 
 
+class PyFlinkBlinkStreamTableTestCase(PyFlinkTestCase):
+    """
+    Base class for stream unit tests.
+    """
+
+    def setUp(self):
+        super(PyFlinkBlinkStreamTableTestCase, self).setUp()
+        self.env = StreamExecutionEnvironment.get_execution_environment()
+        self.env.set_parallelism(1)
```

Comment:
Why we set parallelism to 1 here. It will affect all test case for under blink streaming. Can we set the parallelism in the tests which require that parallelism 1?